lib/powerpoint.js

```diff
@@ -142,6 +167,91 @@ class Powerpoint extends EventEmitter2 {
   }
 
   /**
+   * Acts on image files in place.
+   *
+   * If the last page of a PDF did not represent a full page in the DOM it is
+   * possible for there to be a boundary that has a transparency level that
+   * results in a horizontal line being visible when the image is added to the
+   * powerpoint slide.
+   *
+   * This function crops this transparency and removes the line, by invoking
+   * ImageMagick
+   *
+   * @param sortedImages
+   *
+   * @returns {Promise} when all conversions have been done, and events emitted
+   * Any errors will be logged, but a rejection will not occur.  It is better to get
+   * an output with a line than no output at all.
+   *
+   * @private
+   */
+  _cropLastImages (sortedImages) {
```

Comment:
Yeah it's a hack, but for a situation like this I don't think you had any other options